sql/moz-fx-data-shared-prod/firefox_ios_derived/firefox_ios_clients_v1/checks.sql

```diff
@@ -12,6 +12,23 @@ SELECT
     NULL
   )
 FROM
-  `moz-fx-data-shared-prod.firefox_ios_derived.firefox_ios_clients_v1`
+  `{{ project_id }}.{{ dataset_id }}.{{ table_name }}`
 WHERE
   first_seen_date = @submission_date;
+#warn
+WITH base AS (
```

Comment:
This check is currently failing for all dates. Do we need to add an additional condition here?

Reply:
This is why for now I marked it as a warning. These numbers should match. So will need to investigate further what is causing the discrepancy.